content/posts/fleek-network-running-a-node-in-a-docker-container.md

```diff
@@ -304,13 +304,13 @@ Respectively, execute `docker-build` to build the Docker image:
 make docker-build
 ```
 
-Execute the `docker-run` to run the Docker container based in the built image:
+Execute the `docker-run` to run the Docker container based on the built image:
 
 ```sh
 make docker-run
 ```
 
-⚠️ At the time of writing the `make docker-run` does not persist the Ursa configuration ([options, key store, identity](#fleek-network-managing-the-key-store), etc), you'd be better off executing the actual `docker run` command without the sugar coating for better control and sanity check! This happens because the `-v <HOST-PATH>:<CONTAINER-PATH>` bind mount is missing in the makefile declaration `make docker-run`, this can change at any time so be wary of commands that create unnecessary abstractions! On the other hand, you might find useful to use our [proposed stack](#running-a-stack-with-docker-compose) which has the appropriate binding mounts for you.
+⚠️ At the time of writing the `make docker-run` does not persist the Ursa configuration ([options, key store, identity](#fleek-network-managing-the-key-store), etc), you'd be better off executing the actual `docker run` command without the sugar coating for better control and sanity check! This happens because the `-v <HOST-PATH>:<CONTAINER-PATH>` bind mount is missing in the makefile declaration `make docker-run`, this can change at any time so be wary of commands that create unnecessary abstractions! On the other hand, you might find it useful to use our [proposed stack](#running-a-stack-with-docker-compose) which has the appropriate binding mounts for you.
```

Comment:
should we just remove mentions of `make docker-*` altogether?

Reply:
Sure, later on or something